lsp/server/src/utils/orgUtils.ts

```diff
@@ -182,15 +148,15 @@ export class OrgUtils {
         return globalResult.sobjects.map((sobjetResult) => sobjetResult.name);
     }
 
-    // Retrieves objectInfo folder path, which is '<projectRoot>/.sf/orgName/objectInfos/'
+    // Retrieve objectInfo folder path, which is '<projectRoot>/.sfMobile/orgName/objectInfos/'
     private static objectInfoFolderPath(): string {
         const projectPath = WorkspaceUtils.getWorkspaceDir();
         if (this.orgName === undefined || this.orgName.length === 0) {
             throw new Error('AuthError: No Org exists');
         }
         const objectInfoFolder = path.join(
             projectPath,
-            this.sfFolder,
```

Comment:
Now I'm not sure if `sfFolder` is correct. If the user is using `sfdx` instead of `sf` wouldn't things be saved under `.sfdx` folder instead of `.sf`?

The assumption made here that it is always `.sf` will return a biased path when `objectInfoFolderPath()` is called.

Reply:
We  use '.sfMobile' dir instead.